package.json

```diff
@@ -32,10 +32,13 @@
     "chai": "^4.2.0",
     "commitlint": "^7.6.1",
     "husky": "^2.3.0",
+    "jsdom": "^15.1.1",
+    "jsdom-global": "^3.0.2",
     "mocha": "^6.1.4",
     "mocha-junit-reporter": "^1.22.0",
     "nyc": "^14.1.1",
     "prettier": "^1.17.1",
+    "rxjs": "^6.5.2",
```

Comment:
Shouldn't rxjs be removed from here if it's on peerdeps?

Reply:
nope, it is required in tests. For production-mode (e.g. when you consume it as a library, it exposes its dependncy to rxjs as a peerDep)